check_all_nim_files.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
See #21. I prefer Nim Script tasks which are more cross-platform and easier to maintain for Nim developers.